plugins/aea-ledger-solana/aea_ledger_solana/solana.py

```diff
@@ -429,14 +429,16 @@ def _try_send_signed_transaction(
             )
         stxn = self.deserialize_tx(tx=tx_signed)
         txn_resp = self._api.send_raw_transaction(bytes(stxn.serialize()))
-        retries = 2
+        retries = 15
```

Comment:
2 seconds was too little in my experience

Reply:
Should this not be a method parameter (can be default)

Reply:
Actually, the `_try_send_signed_transaction` method is implementation is wrong. It's only supposed to send the transaction and return the tx digest, but instead it waits for the transaction to settle and returns the tx receipt. I suggest we fix this in the next iteration, so the settlement can be handled by the skill not the ledger.

Reply:
@0xArdi Can you take a look at this PR https://github.com/valory-xyz/open-aea/pull/710

Reply:
I suggest we also add the suggestion by @DavidMinarsch on the next release.